app/views/shared/_cancel_action_modal.html.slim

```diff
@@ -9,6 +11,16 @@
         p.mb1.bold
           = t("#{app_flow}.cancel.warning_header")
         ul.px2.yellow-dots
+          li.mb2
+            - if sp_name
+              - sp_name_tag = content_tag(:span, sp_name, class: 'bold')
+              - warning_point_text = t("#{app_flow}.cancel.warning_point",
+                sp_name: sp_name_tag, warning_qualifier: warning_qualifier)
+              = safe_join(split_tag(split_tag(warning_point_text, sp_name_tag), warning_qualifier))
+            - else
+              - warning_point_text = t("#{app_flow}.cancel.warning_point_no_sp",
+                warning_qualifier: warning_qualifier)
+              = safe_join(split_tag(warning_point_text, warning_qualifier))
```

Comment:
What do you think about moving these variables to methods inside each of the SessionDecorator and ServiceProviderSessionDecorator classes? That way, all you need to do in the view is call `decorated_session.warning_point_text` (for example), and the logic will live in the decorator classes. Let me know if you want to pair on this.